pkg/daemon/ceph/client/command.go

```diff
@@ -134,39 +134,17 @@ func (c *CephToolCommand) run() ([]byte, error) {
 			// Kubectl commands targeting the toolbox container generate a temp
 			// file in the wrong place, so we will instead capture the output
 			// from stdout for the tests
-			if c.timeout == 0 {
-				output, err = c.context.Executor.ExecuteCommandWithOutput(command, args...)
-			} else {
-				output, err = c.context.Executor.ExecuteCommandWithTimeout(c.timeout, command, args...)
```

Comment:
I believe there are other side effects of removing this call to `ExecuteCommandWithTimout()` as explained in the other comment.

Reply:
OK, I'll re-read this code and will send another PR with enough explanation if this code can really be changed.